src/components/home/HomeFooter.jsx

```diff
@@ -0,0 +1,76 @@
+import styled from 'styled-components';
+import { icTravel, icWish, icSearch, icMassage, icProfile } from 'assets';
+import { Link } from 'react-router-dom';
+
+function NavBar() {
+  return (
+    <StyledFooter>
+      <StyledNavToggle>
+        <img src={icSearch} />
+        <span>둘러보기</span>
+      </StyledNavToggle>
+      <Link to={'./Wish'}>
+        <StyledNavToggle>
+          <img src={icWish} />
+          <span>위시리스트</span>
+        </StyledNavToggle>
+      </Link>
+      <Link to={'./'}>
```

Comment:
여기도 마찬가지로 { } 필요 없고 그냥 "/"만 해주면 돼!
```suggestion
      <Link to="/">
```